en/flight_modes_rover/mecanum.md

```diff
@@ -0,0 +1,128 @@
+# Drive Modes (Mecanum Rover)
+
+Flight modes (or more accurately "Drive modes" for ground vehicles) provide autopilot support to make it easier to manually drive the vehicle or to execute autonomous missions.
+
+This section outlines all supported drive modes for mecanum rovers.
+
+For information on mapping RC control switches to specific modes see: [Basic Configuration > Flight Modes](../config/flight_mode.md).
+
+::: warning
+Selecting any other mode than those listed below will either stop the rover or can lead to undefined behaviour.
+:::
+
+## Manual Modes
+
+Manual modes require stick inputs from the user to drive the vehicle.
+
+![Manual Controls](../../assets/airframes/rover/flight_modes/manual_controls_mecanum.png)
```

Comment:
I am confused by this image, and possibly by the equivalent images in the other rovers.

The question is "how does the image on the left map to the movement on the right?" or "there is no way to know from the image how the vehicle is actually going to move based on the sticks"

For example: 

On the left stick we have label U/D
- I think should be F/B. I presume these control speed, not acceleration? We should say that on the text.
Left stick L/R are yaw right?
-  They should label the movement they control and ideally color co-ordinate with the image on the right - i.e. so if you show a movement on a particular stick on the left that should match on the image on the right.

<img width="614" alt="image" src="https://github.com/user-attachments/assets/ab8d7141-a5f8-46e5-8dc4-be95d881fb7b">

Reply:
Changed in https://github.com/PX4/PX4-user_guide/pull/3411/commits/3cb0729a3821c928048d3c9c2d99f12ac0501041